modules/ebi-metagenomics/samtools/bam2fq/main.nf

```diff
@@ -29,8 +29,8 @@ process SAMTOOLS_BAM2FQ {
             -@ $task.cpus \\
             -1 ${prefix}_1.fq.gz \\
             -2 ${prefix}_2.fq.gz \\
-            -0 ${prefix}_other.fq.gz \\
-            -s ${prefix}_singleton.fq.gz \\
+            -0 /dev/null \\
```

Comment:
I think we should keep those files, someone may need them. I propose we split the outputs:
```
    output:
    tuple val(meta), path("*{_1,_2}.fq.gz"),      optional: true, emit: reads
    tuple val(meta), path("*_interleaved.fq.gz"), optional: true, emit: interleaved_reads
    tuple val(meta), path("*_singleton.fq.gz"),   optional: true, emit: singleton_reads
    tuple val(meta), path("*_other.fq.gz"),       optional: true, emit: other_reads
    path "versions.yml",                                          emit: versions
```
or even merging _1,_2 and _interleaved:
```
    output:
    tuple val(meta), path("*{_1,_2,_interleaved}.fq.gz"),         emit: reads
    tuple val(meta), path("*_singleton.fq.gz"),   optional: true, emit: singleton_reads
    tuple val(meta), path("*_other.fq.gz"),       optional: true, emit: other_reads
    path "versions.yml",                                          emit: versions
```